assets/src/dashboard/components/cardGallery/components.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+export const GalleryContainer = styled.div`
+  ${({ maxWidth }) => `
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+    max-width: ${maxWidth}px;
+  `}
+`;
+
+export const MiniCardsContainer = styled.div`
+  ${({ rowHeight, gap }) => `
+    display: grid;
+    grid-template-columns: auto auto auto;
```

Comment:
nit: you can do `repeat(3, auto)` and get the same thing

Reply:
I knew there had to be a shortcut to do all 3, I tried googling but couldn't get the phrasing right.  Thanks for the tip!